src/app/electra/action-names.ts

```diff
@@ -5,3 +5,7 @@ export const INITIALIZE_ELECTRA_FAIL = 'INITIALIZE_ELECTRA_FAIL'
 export const GENERATE_HARD_WALLET = 'GENERATE_HARD_WALLET'
 export const SUCCESSFULLY_GENERATED_HARD_WALLET = 'SUCCESSFULLY_GENERATED_HARD_WALLET'
 export const FAILED_T0_GENERATE_HARD_WALLET = 'FAILED_T0_GENERATE_HARD_WALLET'
+
+export const GET_GLOBAL_BALANCE = 'GET_GLOBAL_BALANCE'
+export const GET_GLOBAL_BALANCE_SUCCESS = 'GET_GLOBAL_BALANCE_SUCCESS'
```

Comment:
Keep the constants consistent with the rest, e.g.

Instead of `GET_GLOBAL_BALANCE_SUCCESS`, `SUCCESSFULLY_RETRIEVED_GLOBAL_BALANCE`

or change `SUCCESSFULLY_GENERATED_HARD_WALLET` and the fail one to a syntax similar to yours.